types/jasmine-enzyme/jasmine-enzyme-tests.tsx

```diff
@@ -254,7 +254,7 @@ describe('toHaveRef', () => {
 describe('toHaveState', () => {
 	class Fixture extends React.Component {
 		constructor() {
-			super();
+			super({});
```

Comment:
I know it's a test, but passing an empty object is not really correct here and later on - people look at tests and learn from them. I'd rather you take props in parameters and pass the object further on.

Reply:
It’s perfectly correct. The type of the props is `{}`; if it wasn’t correct it wouldn’t type check.

Reply:
With `--strictFunctionTypes` - it's correct. But `{}` used to refer to any object, didn't it? Therefore, it's not *perfectly* correct 😄